src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -151,19 +162,50 @@ const Status: React.FC<Pick<Props, 'order' | 'isOverBalance' | 'transactionHash'
   ])
   const isFilled = useMemo(() => isOrderFilled(order), [order])
   // Display isLowBalance warning only for active and partial fill orders
-  const isLowBalance = isOverBalance && !isUnlimited && !isFilled && !isScheduled && !isPendingOrder && !isExpiredOrder
+  const isLowBalance =
+    isOverBalance &&
+    !isUnlimited &&
+    !isFilled &&
+    !isScheduled &&
+    !isActiveNextBatch &&
+    !isPendingOrder &&
+    !isExpiredOrder
 
   const pending = useMemo(() => isPendingOrder && <PendingLink transactionHash={transactionHash} />, [
     isPendingOrder,
     transactionHash,
   ])
 
+  // Dima's trick to force component update
+  const [, forceUpdate] = useSafeState({})
+
+  // Updates the component every second, if it has one of the states where it should happen
```

Comment:
Wouldn't you rather update after batch ended instead of rerendering every second
```
setTimeout(..., (BATCH_TIME - secondsRemaining + 1) * 1000) // to fall on after the batch ended 
//and then again for every batch end
setInterval(..., BATCH_TIME * 1000)
```

Take [useTimeRemainingInBatch](https://github.com/gnosis/dex-react/blob/ead3ff4e2e60d1d69bc2b2425ff116e34d68e4be/src/hooks/useTimeRemainingInBatch.ts#L57) for reference

And while you are at it can you change ~window.~ `setTimeout|setInterval` in there. I had problems with TS and id: NodeJS.Timer, but here I see TS didn't complain for you. Please :pleading_face:

Reply:
But but, to render the countdown, I'll need to refresh every second, no?

Reply:
Oh, shit, I didn't think about render
Yep, that's true
Hate that we have so much to display

Reply:
> And while you are at it can you change window. setTimeout|setInterval in there. I had problems with TS and id: NodeJS.Timer, but here I see TS didn't complain for you. Please pleading_face

It didn't complain because of `if (timer) clearTimeout(timer)`

If I apply your suggestion from https://github.com/gnosis/dex-react/pull/930#discussion_r410036030, then the typing goes to shit.

I'd rather keep the `if (timer)` and not have TS annoyances

Reply:
Check the latest verison